src/backend/services/users-api/src/app/routes.py

```diff
@@ -0,0 +1,114 @@
+from src.app import app
+from flask import request, jsonify
+from src.app import mongo
+import sys, os
+
+if os.environ.get('SERVICE_IN_DOCKER', False):
+    sys.path.append(os.path.abspath(os.path.join('..', '')))
+else:
+    sys.path.append(os.path.abspath(os.path.join('../..', '')))
+
+from shared import HttpCode, FailureReturnString, config
+
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + mongo.db.name
+    )
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    data = request.get_json(force=True)
+
+    try:
+        username = data['username']
+        password = data['password']
+    except Exception as e:
+        return jsonify(err= FailureReturnString.INCORRECT_PAYLOAD.value), HttpCode.BAD_REQUEST.value
+
+    user = mongo.db.users.find_one({"username":username})
+
+    return jsonify(success = (user["password"] == password))
+
+@app.route("/getUserInfo", methods=["POST"])
+def getUserInfo():
```

Comment:
The route should be GET /<username>.

Reply:
Agreed with the above, except umnetId not username (if we're explicitly putting the name in the route)

Reply:
Shouldn't that stay username @ZackHolmberg @abhisachdev17? I agree that we're using the UMNetIDs as the username but as far as the app context is concerned, it is just a unique username.

Reply:
I'll just use this thread to discuss this topic instead of other similar ones:

I like `umnetId` because it's explicit, and since we don't have a general username and we are a UofM specific thing I feel it doesn't hurt to just explicitly call it `umnetId`, since the field will never be anything else but a `umnetId`. this way, when we are retrieving data, or if a third party developer was retrieving data, they won't get confused on what value they are getting as they might not know that `username` == `umnetId` like we do, and as such, they might get confused and not hit the endpoint if they are trying to get the `umnetId`. We don't really need to abstract it in our backend because who are we abstracting it from? I think in most cases it's better to be explicit about data consumption rather than abstract it, but that's just my opinion! What do you folks think?

Reply:
Those are some good points. I was under the impression of a third-party app wanting to use our services who might not have access to `umnetid`. But if we only want to say that the third-party apps should be connected to `umnetid` then it makes sense to have that.

Reply:
Changed to umnetID